src/emitfromevents.cpp

```diff
@@ -116,8 +116,13 @@ void EmitFromEvents::BeginNode() {
 }
 
 void EmitFromEvents::EmitProps(const std::string& tag, anchor_t anchor) {
-  if (!tag.empty() && tag != "?" && tag != "!")
-    m_emitter << VerbatimTag(tag);
+  if (!tag.empty() && tag != "?" && tag != "!"){
+      if (tag[0] == '!') {
```

Comment:
Sorry I'm late to this, but could you fix the indentation here? The codebase uses 2 spaces for indentation, but there is 4 here. Thanks in advance.

Reply:
Can you just submit a new PR? Agreed, this PR should have had clang-format run.